.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -73,6 +78,7 @@ Please make sure the items below have been covered before requesting a review:
 - [ ] This change has relevant unit tests (if applicable).
 - [ ] This change has relevant documentation additions / updates (if applicable).
 - [ ] (For Automatticians) I've created a changelog draft. 
+- [ ] (For Automatticians) I've received approval for this change / I'm performing a proposal-exempt action.
```

Comment:
Not sure if this is necessary given the new section above. I mainly worry about the checklist becoming super long and tedious to keep up with.

Reply:
I'm fine with removing it. 

The only thing it's potentially useful for is to reiterate the process flow and prevent possible "you need to go to upstream first" rejections. But I'm not even sure it would do a whole lot of prevent that honestly.

Reply:
Removed.